apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PriorityList.cs

```diff
@@ -22,10 +22,14 @@ namespace Google.Cloud.Spanner.V1
     /// A simple sorted list that sorts based on the result of CompareTo,
     /// but also allows duplicate entries based on CompareTo.
     /// </summary>
-    internal class PriorityList<T> where T : IPriorityListItem<T>
+    /// <remarks>This is not intended for general purpose use; it is designed for use with Spanner clients.</remarks>
+    public class PriorityList<T> where T : IPriorityListItem<T>
```

Comment:
SGTM. I'd been wondering about creating an Internal namespace. Glad we're thinking alike.
Will do that in a second commit in this PR.